src/ContactPeople/ContactPeopleForm.js

```diff
@@ -68,6 +69,10 @@ class ContactPeopleForm extends Component {
     return `${starting}${ending}`;
   }
 
+  addContacts = contact => {
```

Comment:
could you please return array already? I think we will come up to this in a future and it's not hard to handle array for now, is it?

Reply:
yep, it's ok